pkgs/applications/audio/giada/default.nix

```diff
@@ -50,11 +64,18 @@ stdenv.mkDerivation rec {
     libXcursor
     catch2
     nlohmann_json
+    flac
+    libogg
+    libvorbis
+    libopus
+    rtmidi
   ];
 
   postPatch = ''
     sed -i 's:"deps/json/single_include/nlohmann/json\.hpp":<nlohmann/json.hpp>:' \
         src/core/{conf,init,midiMapConf,patch}.cpp
+    sed -i 's:<RtMidi.h>:<${rtmidi.src}/RtMidi.h>:' \
+        src/{core/kernelMidi,gui/elems/config/tabMidi,utils/ver}.cpp
```

Comment:
Can we use substituteInPlace instead? Also please use / or | as a delimiter for sed.